docs/src/main/sphinx/release/release-377.md

```diff
@@ -7,9 +7,10 @@
 
 ## Hive connector
 
-* Add support for partition names that include special characters. ({issue}`11719`)
 * Add support for `date` type partition names with timestamp formatting. ({issue}`11873`)
 * Improve performance of queries that use Glue metadata. ({issue}`11869`)
+* Fix `sync_partition_metadata` procedure when partition names differ from
```

Comment:
`sync_partition_metadata` can cope with partition names which differ from the partition paths on the file system

Reply:
Thats not a full sentence .. also still missing what this did before

E.g. 

Prevent failures of ...

Reply:
Well... still not sure what happened before.. did data get corrupted or did the procedure just fail or something else?

Reply:
Due to the fact that the comparison mechanism between the partition names stored in HMS and partition paths on the file system/ object storage was incorrect, when trying to sync the partition metadata the second time for a table, the user kept hitting rather puzzling exceptions. 

When using 'FULL' mode after the first successful 'FULL' mode run:

```
dropping a partition added in the same transaction is not supported: testschema.test_table_external [2021-04-25, 20:00:13, 12345]
```
When using 'ADD' mode after the first successful 'FULL' mode run :

```
One or more partitions already exist for table 'testschema.test_table_external'
```

This was due to the fact that partition paths corresponding to existing partition names in HMS were falsely marked to be added/removed from HMS, reason why HMS threw the exceptions mentioned earlier. 

I hope this comment gives a better context of the problem.

Reply:
Okay .. thats rather tricky. Do you have any suggestions on how to make a useful release notes entry out of this @martint ?

Maybe just 

Prevent failures of the `sync_partition_metadata` system procedure for inconsistent partition names stored in Hive metastore and used in the file system.